CONTRIBUTING.md

```diff
@@ -77,7 +77,7 @@ It is possible to develop the Ansible roles without needing to build the collect
   vars:
     cluster_type: rosa
     cluster_name: fvtrosa
-    ocp_version: "4.10.17"
+    ocp_version: "4.14.5"

```

Comment:
i think we have moved to 4.14.12

Reply:
used latest 4.14.16

Reply:
we are using the latest 4.14.16.. Shall we keep it to 4.14.12.. thats what we have tested in fvt and in airgap too